### PR TITLE
adding release notes for 0.0.3

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

File: docs/source/release_notes.rst
Comment (kc611) [inline docs/source/release_notes.rst:9]:
```suggestion
an optional dependency in Numba 0.58.
```

**Comment by esc** [comment]
@kc611 thank you for the review this has now been updated. I also added #97 to the change-log as that has been merged to `main` and is only waiting on it's backport at #100 to be merged. This will not need a rebase and can be merged after #100

**Review by kc611 — APPROVED** [review]
LGTM!

**Comment by esc** [comment]
@kc611 thank you for the review.

**Pull request merged**
